pipenv/routines/requirements.py

```diff
@@ -0,0 +1,78 @@
+import re
+import sys
+
+from pipenv.utils.dependencies import get_lockfile_section_using_pipfile_category
+from pipenv.vendor import click
+
+
+def requirements_from_deps(deps, include_hashes=True, include_markers=True):
+    pip_packages = []
+
+    for package_name, package_info in deps.items():
+        # Handling git repositories
+        if "git" in package_info:
+            git = package_info["git"].replace("ssh://", "https://")
+            ref = package_info.get("ref", "")
+            extras = (
+                "[{}]".format(",".join(package_info.get("extras", [])))
+                if "extras" in package_info
+                else ""
+            )
+            pip_package = f'"{package_name}{extras} @ git+{git}@{ref}"'
```

Comment:
```suggestion
            pip_package = f'{package_name}{extras} @ git+{git}@{ref}'
```

Enclosing quotes causes pip to fail with:

```
ERROR: pyjwt @ git+https://git@github.com/jpadilla/pyjwt.git@7665aa625506a11bae50b56d3e04413a3dc6fdf8 is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+ht
  1 -i https://pypi.org/simple
tp, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
```